### PR TITLE
Added more pagination options for /devicesview endpoint and updates s…

### PR DESCRIPTION
**Pull request by AaronH88**
…pec file.

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [ ] Refactor

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] I run `go fmt ./...` to check that my code is properly formatted
- [X] I run `go vet ./...` to check that my code is free of common Go style mistakes

### CODE REVIEW

File: pkg/routes/devices.go [hunk 1]
```diff
@@ -71,6 +71,14 @@ var devicesFilters = common.ComposeFilters(
 		QueryParam: "update_available",
 		DBField:    "devices.update_available",
 	}),
+	common.ContainFilterHandler(&common.Filter{
+		QueryParam: "status",
+		DBField:    "devices.status",
+	}),
+	common.ContainFilterHandler(&common.Filter{
+		QueryParam: "image_id",
+		DBField:    "devices.imageid",
```
Comment (ldjebran) [inline pkg/routes/devices.go:80]:
```suggestion
		DBField:    "devices.image_id",
```

File: pkg/routes/devices.go [hunk 2]
```diff
@@ -71,6 +71,14 @@ var devicesFilters = common.ComposeFilters(
 		QueryParam: "update_available",
 		DBField:    "devices.update_available",
 	}),
+	common.ContainFilterHandler(&common.Filter{
+		QueryParam: "status",
+		DBField:    "devices.status",
```
Comment (ldjebran) [inline pkg/routes/devices.go:76]: we have no status in devices model

Comment (AaronH88) [reply pkg/routes/devices.go]: Thanks, removed it

**Review by ldjebran — APPROVED** [review]
LGMT

**Pull request merged**